hackathon_site/registration/views.py

```diff
@@ -93,6 +93,11 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class ApplicationView(LoginRequiredMixin, FormView):
+class ApplicationView(LoginRequiredMixin, ModelFormMixin, FormView):
     form_class = ApplicationForm
     template_name = "registration/application.html"
+
+    def get_form_kwargs(self):
```

Comment:
Also what does this do

Reply:
https://docs.djangoproject.com/en/3.1/ref/class-based-views/mixins-editing/#django.views.generic.edit.FormMixin.get_form_kwargs